dpnp/dpnp_iface_arraycreation.py

```diff
@@ -2738,17 +2679,15 @@ def tril(m, /, *, k=0):
         _k = operator.index(k)
     except TypeError:
         pass
+    if _k is None:
+        raise TypeError(f"`k` must be a integer data type, but got {type(k)}")
 
-    if not isinstance(m, (dpnp.ndarray, dpt.usm_ndarray)):
-        pass
-    elif m.ndim < 2:
-        pass
-    elif _k is None:
-        pass
-    else:
-        return dpnp_container.tril(m, k=_k)
+    dpnp.check_supported_arrays_type(m)
 
-    return call_origin(numpy.tril, m, k)
+    if m.ndim < 2:
+        raise ValueError("Input array must have 2 or more dimensions")
```

Comment:
could you please create an issue to get rid of the limitation, at least for `m.ndim == 1`